lib/video_info/providers/youtubeplaylist_scraper.rb

```diff
@@ -0,0 +1,27 @@
+require 'nokogiri'
+require 'open-uri'
+require 'open_uri_redirections'
+
+class VideoInfo
+  module Providers
+    module YoutubePlaylistScraper
+      def description
+	data.css('meta').find { |m| m.values[0] == 'description' }.values[1]
+      end
+      
+      def title
+	data.css('meta').find { |m| m.values[0] == 'title' }.values[1]
```

Comment:
Prefer `detect` over `find`.<br>Use 2 (not -5) spaces for indentation.<br>Tab detected.